internal/resource/mappers/bimapper/bimapper_test.go

```diff
@@ -131,10 +133,13 @@ func TestMapper(t *testing.T) {
 	requireServicesTracked(t, m, wwwSvc)
 
 	// track new one that overlaps slightly
-	m.TrackItem(fail2.Id, fail2_refs)
+	m.TrackItem(fail2.Id, fail2Refs)
+
+	// Check links mapping for the new one
+	requireLinksForItem(t, m, fail2.Id, fail2Refs...)
```

Comment:
It looks like this line is duplicated a few lines down. Is that intended?

Reply:
Great catch! I think this was an unhappy rebase